sources/docker-compose.yml

```diff
@@ -149,6 +149,7 @@ services:
       - ESDC_REQUEST_FOLDER=${ESDC_REQUEST_FOLDER}
       - ESDC_RESPONSE_FOLDER=${ESDC_RESPONSE_FOLDER}
       - ESDC_ENVIRONMENT_CODE=${ESDC_ENVIRONMENT_CODE}
+      - APPLICATION_ARCHIVE_DAYS=${APPLICATION_ARCHIVE_DAYS}
```

Comment:
can we remove APPLICATION_ARCHIVE_DAYS it from api environments? and also remove from api-deploy.yml?